### ISSUE TITLE
Suspend plugin 404

### ISSUE THREAD

**Issue by CyberCyclone** [remcoder/fix-back-button-android#1]
The dependancy for the Suspend plugin returns 404, which breaks the installation of this plugin.

https://github.com/Lamerchun/org.android.tools.suspendXNH8u00gSjlIeIDg - returns 404.

**Comment by tripflex** [comment]
Yeah weird i had the same issue, had to manually install it using `meteor add cordova:org.android.tools.suspend@0.1.2` and worked fine ... weird

**Comment by tripflex** [comment]
Correction ... it doesn't work with just the above command, I had to actually use the specific reference to github since it does not appear to exist anymore on npm.

So instead, just install it like this:
```
meteor add cordova:org.android.tools.suspend@https://github.com/Lamerchun/org.android.tools.suspend.git#0dbb52cca0244ba22a8c7975895f0f45d2e9a4a9
```
